extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -0,0 +1,51 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.scrollHandler', [
+    'ext.wikia.adEngine.adContext',
+    'wikia.log',
+    'wikia.document',
+    'wikia.window'
+], function (adContext, log, doc, win) {
+
+    var logGroup = 'ext.wikia.adEngine.slot.scrollHandler',
+        isRefreshed = {
+            PREFOOTER_LEFT_BOXAD: false,
+            PREFOOTER_RIGHT_BOXAD: false
+        };
+
+    function init() {
+        if (adContext.getContext().opts.enableScrollHandler) {
+            win.addEventListener('scroll', function () {
```

Comment:
We could also add some throttling here, so it fires the function every `200ms`. It would make performance of our page better. What do you think?